packages/darklang/stdlib/string.dark

```diff
@@ -56,6 +106,68 @@ module Darklang =
       let fromChar (c: Char) : String = PACKAGE.Darklang.Stdlib.Char.toString c
 
 
+      /// URLBase64 encodes a string without padding. Uses URL-safe encoding with
+      /// {{-}} and {{_}} instead of {{+}} and {{/}}, as defined in
+      /// [RFC 4648 section 5](https://www.rfc-editor.org/rfc/rfc4648.html#section-5)
+      let base64Encode (s: String) : String = Builtin.String.base64Encode s
+
+
+      /// Base64 decodes a string. The returned value is wrapped in {{Result}}.
+      /// Works with both the URL-safe and standard Base64
+      /// alphabets defined in [RFC 4648
+      /// sections](https://www.rfc-editor.org/rfc/rfc4648.html)
+      /// [4](https://www.rfc-editor.org/rfc/rfc4648.html#section-4) and
+      /// [5](https://www.rfc-editor.org/rfc/rfc4648.html#section-5).
+      let base64Decode
+        (s: String)
+        : PACKAGE.Darklang.Stdlib.Result.Result<String, String> =
+        Builtin.String.base64Decode s
+
+
+      /// Take a string and hash it to a cryptographically-secure digest.
+      /// Don't rely on either the size or the algorithm.
+      let digest (s: String) : String = Builtin.String.digest s
```

Comment:
This feels a little off to have as a builtin. Can it be done using Bytes or directly call the hashing function?